bundles/AdminBundle/public/js/pimcore/document/pages/settings.js

```diff
@@ -20,64 +20,6 @@ pimcore.document.pages.settings = Class.create(pimcore.document.settings_abstrac
     getLayout: function () {
 
         if (this.layout == null) {
-            // meta-data
-            var addMetaData = function (value) {
-
-                if(typeof value != "string") {
-                    value = "";
-                }
-
-                var count = this.metaDataPanel.query("button").length+1;
-
-                var compositeField = new Ext.form.FieldContainer({
-                    layout: 'hbox',
-                    hideLabel: true,
-                    items: [{
-                        xtype: "textfield",
-                        value: value,
-                        width: 636,
-                        name: "metadata_" + count,
-                    }]
-                });
-
-                compositeField.add({
-                    xtype: "button",
-                    iconCls: "pimcore_icon_delete",
-                    handler: function (compositeField, el) {
-                        this.metaDataPanel.remove(compositeField);
-                        this.metaDataPanel.updateLayout();
-                    }.bind(this, compositeField)
-                });
-
-                this.metaDataPanel.add(compositeField);
-                this.metaDataPanel.updateLayout();
-            }.bind(this);
-
-            this.metaDataPanel = new Ext.form.FieldSet({
-                title: t("html_tags") + " (&lt;meta .../&gt; &lt;link .../&gt; ...)",
-                collapsible: false,
-                autoHeight:true,
-                width: 700,
-                style: "margin-top: 20px;",
-                items: [{
-                    xtype: "toolbar",
-                    style: "margin-bottom: 10px;",
-                    items: ["->", {
-                        xtype: 'button',
-                        iconCls: "pimcore_icon_add",
-                        handler: addMetaData
-                    }]
-                }]
-            });
-
-            try {
-                if(typeof this.document.data.metaData == "object" && this.document.data.metaData.length > 0) {
-                    for(var r=0; r<this.document.data.metaData.length; r++) {
-                        addMetaData(this.document.data.metaData[r]);
-                    }
-                }
-            } catch (e) {}
-
 
```

Comment:
There is a leftover here

https://github.com/pimcore/pimcore/blob/113e29bd934c3ed54be6666498c75f71ef4d6537/bundles/AdminBundle/public/js/pimcore/document/pages/settings.js#L119

Reply:
Done in [2f53525](https://github.com/pimcore/pimcore/pull/14411/commits/2f535259776aac8efdc2685d1aaeca404d3eb732)